lib/payload/scanner/header_xss/engine.py

```diff
@@ -13,26 +13,34 @@ def header_xss(host, payloads_xss = None, headers_xss = None):
     Return
         Result : True or False
     '''
+    if data is None:
+        data = {'a': '1'}
     if payloads_xss is None:
-        payloads_xss = '<script>alert(/1/);</script>'
+        payloads_xss = r'<script>alert()</script>'
     if headers_xss is None:
         headers_xss = {
-                'User-Agent': payloads_xss
-                ,'Except': payloads_xss
-                ,'Cookie': payloads_xss
-                ,'Referer': payloads_xss
-                ,'Accept-Encoding': payloads_xss
-                ,'Accept-Language': payloads_xss
-                ,'Accept': payloads_xss
-                }
+                'User-Agent'        : payloads_xss,
+                'Except'            : payloads_xss,
+                'Cookie'            : payloads_xss,
+                'Origin'            : payloads_xss,
+                'Referer'           : payloads_xss,
+                'Accept-Encoding'   : payloads_xss,
+                'Accept-Language'   : payloads_xss,
+                'Accept'            : payloads_xss,
+                'X-Forwaded-For'    : payloads_xss
+            }
     try:
-        r = requests.head(host)
-        for header in r.headers:
-            headers_xss[header] = payloads_xss
-        req = requests.post(host, headers=headers_xss)
-        if payloads_xss.lower() in req.text.lower():
-            return True
-        else:
-            return False
+        # Expanding our scope of attack by using all the
+        # HTTP methods available, HEAD and OPTIONS return 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5251812)

Reply:
Hey @Ali-Razmjoo, thanks for the notification! Fixed the trailing whitespace.